ni_measurementlink_service/_internal/utilities/_constants.py

```diff
@@ -0,0 +1,5 @@
+# Constants for Annotations
```

Comment:
Docstring goes here

1st line of a docstring is a summary in sentence case (not Title Case)

Reply:
This is still not a docstring. Please read https://peps.python.org/pep-0257/